### ISSUE TITLE
URL parsing

### ISSUE THREAD

**Issue by ianremsen** [langel/botb_irc_bot#27]
When a URL is posted in chat, displays the title of the page, some contextual information, such as the content of a tweet if it's a url to a tweet, the length/likes/uploader of a youtube video, etc

**Comment by ianremsen** [comment]
On second thought, it seems bots pretty much have it covered in #botb already. Maybe an emergency boolean switch sort of thing?

**Comment by langel** [comment]
I'd really rather our bot handled it and Chromaryu forked over Sylphy's code lol

**Comment by ianremsen** [comment]
Yeah, having auxiliary bots for what really should be a core function seems odd.
